waiter/integration/waiter/saml_authentication_test.clj

```diff
@@ -0,0 +1,111 @@
+(ns waiter.saml-authentication-test
+  (:require [clojure.data.json :as json]
+            [clojure.string :as string]
+            [clojure.test :refer :all]
+            [waiter.util.client-tools :refer :all]
+            [clojure.java.shell :as shell])
+  (:import (java.net URLEncoder)))
+
+(deftest ^:parallel ^:integration-fast test-default-composite-authenticator
```

Comment:
as discussed offline, we should try and avoid the shelling out and replace it with clojure code.

Reply:
done